File: test/fixtures/testsuites/fake_submission_test.exs
Code.load_file("test/fixtures/testsuites/fake_submission.exs")
ExUnit.start

defmodule FakeSubmissionTest do
  use ExUnit.Case, async: true
  doctest FakeSubmission

  test "this passes" do
    assert FakeSubmission.this_is_true == true
  end
  
end

File: test/fixtures/testsuites/fake_submission_test.rb
require 'minitest/autorun'

require File.expand_path('../fake_submission.rb', __FILE__)

class FakeSubmissionTest < Minitest::Test

  def test_this_will_pass
    @fake_submission = FakeSubmission.new
    assert @fake_submission.this_is_true
  end
  
end
